{{cookiecutter.project_slug}}/docs-sphinx/conf.py

```diff
@@ -20,10 +20,10 @@
 import os
 import sys
 
-import {{ cookiecutter.package_slug }}
-
-sys.path.insert(0, os.path.abspath(".."))
+if os.path.exists('src'):
```

Comment:
the user project should be installed locally before running the documentation stuff .. so we don't need to change the sys.path manually at all, just the import should be enough
could you try to remove this `if` and the `sys.path` lines please?

Reply:
That gave me headaches xD